tests/integration/world_io_test.py

```diff
@@ -21,3 +20,36 @@ def testLoadWorld(self):
     def testLoadInvalidWorld(self):
         with self.assertRaises(LoadWorldException):
             GraspitCommander.loadWorld("")
+
+    def testImportObstacle(self):
+        GraspitCommander.importObstacle("floor")
```

Comment:
can you throw in an assert call to verify that the obstacle was placed? 

Maybe use getObstacles and assert the len of the list is only 1 and then use getObstacle to check the obstacle's properties?